packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -31,35 +32,31 @@ However, if an error occurs during sign-in, Okta may need to display an error pa
-
-**In what situations does Okta serve error pages to the user?**
-
-The error page appears when a critical error occurs or an application is misconfigured. See [Okta deployment models &mdash; redirect vs. embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) for more information on Okta-hosted (redirect) functionality.
-
-**What can I customize on the error page?**
-
-You can add any HTML, CSS, or JavaScript that you want to the page.
-
 ### Use the Brands API
 
 The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across your Okta-hosted Sign-In Widget, error pages, email templates, and Okta End-User Dashboard all at once. You don't need to set a customized Okta URL domain. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes) for more information on this feature and how to use it.
 
 ## Edit the error page
 
-The HTML editor that is provided on the **Error pages code editor** section of **Customizations** allows you to modify any HTML, CSS, and JavaScript on the error page.
+Use the code editor to modify any HTML, CSS, or JavaScript on the error page. See [Customization examples](#customization-examples) for snippets that you can update and use.
 
-1. In the Admin Console, select **Customizations** and then **Error pages code editor**.
-1. Make changes directly in the embedded **Error Pages** editor.
-1. Click **Preview** to preview your changes before you publish.
-1. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
-1. Click **Save and Publish** to commit your changes.
+> **Note:** There is no draft mode in the code editor. When you click **Publish**, your changes go live. If you want to test changes without impacting production, use a test or development org.
 
-> **Note:** See [Customization examples](#customization-examples) for snippets that you can update and use.
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Okta-hosted Error Pages box**, click **Edit**.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+4. Make changes directly in the editor. If you enter `{`, `(`, or `.` you see a list of available variables that you can use. See [Use variables](#use-variables).
+   * Click **Preview** to see your changes in a new browser window before you publish.
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
+   * Click **Revert changes** to remove your customizations and restore the default HTML/CSS and JavaScript code.
 
-## Use macros
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, turn off the toggle next to **Code editor**.
```

Comment:
To discard your changes without publishing them, either **Revert changes** or turn off the toggle next to **Code editor** to remove all custom code.

Reply:
Updated